src/main/java/blob/tasks/Task.java

```diff
@@ -0,0 +1,62 @@
+package blob.tasks;
+
+/**
+ * The Task class encapsulates a task.
+ * A task has a string description describing the task to be done, and a boolean indicator to represent
+ * the status of completion of the task.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    protected TaskType type;
+
+    public Task(String description, TaskType type) {
+        this.description = description;
+        this.type = type;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns an icon representing the state of completion of the task.
+     *
+     * @return A checkmark icon (✓) if the task is done, and " " otherwise
+     */
+    public String getStatusIcon() {
```

Comment:
Excellent use of a tick instead of a cross as suggested. Creative and user-friendly.